packages/espressocash_app/lib/features/ramp/services/off_ramp_order_service.dart

```diff
@@ -330,13 +330,12 @@ class OffRampOrderService implements Disposable {
       RampPartner.kado => sl<KadoOffRampOrderWatcher>(),
       RampPartner.scalex => sl<ScalexOffRampOrderWatcher>(),
       RampPartner.coinflow => sl<CoinflowOffRampOrderWatcher>(),
-      RampPartner.moneygram => null,
       RampPartner.rampNetwork ||
       RampPartner.moneygram ||
```

Comment:
Why should it throw on moneygram? Isn't previous PR handling it?

Reply:
Actually I think, this is does not matter, because this method is never reached if RampPartner is moneygram

Reply:
Why?

Reply:
```
  @PostConstruct(preResolve: true)
  Future<void> init() async {
    final query = _db.select(_db.offRampOrderRows)
      ..where(
        (tbl) => tbl.status.isNotInValues([
          OffRampOrderStatus.completed,
          OffRampOrderStatus.cancelled,
        ]),
      );

    final orders = await query.get();

    for (final order in orders) {
      if (order.partner == RampPartner.moneygram) {
        continue;
      }

      _subscribe(order.id);
      unawaited(_watch(order.id));
    }
  }
```

Reply:
Same as in on_ramp_order_service, we use another services (MoneygramOnRampOrderService/MoneygramOffRampOrderService ) to subscribe to moneygram order.

Reply:
Ok then, let's have it with throwing then, just leave the comment there. And we'll need to rethink this mechanism in the future, it's confusing.